test/pls.jl

```diff
@@ -501,6 +515,25 @@ end
         @test bic(fm) ≈ bic(m)
         @test coef(fm) ≈ coef(m)
     end
+    @testset "profile" begin
+        pr = profile(last(models(:sleepstudy)))
```

Comment:
I got a warning here:

```
┌ Warning: Reverse spline for θ3 is not monotone.
```

Reply:
That warning is expected.  It is tricky to ensure that the reverse spline is monotone.  I suggest leaving that in there so that we know we have an issue to address.